next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.types.ts

```diff
@@ -34,6 +34,20 @@ export interface CldUploadWidgetProps {
   options?: CloudinaryUploadWidgetOptions;
   signatureEndpoint?: URL | RequestInfo;
   uploadPreset?: string;
+  onSuccessAction?: CldUploadEventAction;
+  onUploadAction?: CldUploadEventAction;
```

Comment:
the `onUpload` function is deprecated and will be removed in a later version (likely a major?), so thinking we probably shouldnt add this as an option. while it will technically work if someone tries because the actions are automatically created, we don't want to document or promote its use as a valid option only for it to be pulled out soon